data/publiccloud/pcw/sshd_config

```diff
@@ -0,0 +1,3 @@
+PubkeyAuthentication yes
+PermitRootLogin yes
+AuthorizedKeysFile .ssh/authorized_keys
```

Comment:
We use keys and that should be permitted by default. This file is not needed.

Reply:
> Neither of those options is needed. We log in to Larry as `root` via ssh key and look:
> 
> ```shell
> publiccloud-ng:~ # cat /etc/ssh/sshd_config
> cat: /etc/ssh/sshd_config: No such file or directory
> ```

They are needed. Look at `/etc/ssh/sshd_config.d/sshd_config.conf`, after usrmerge one should not edit `/etc/ssh/sshd_config` directly but place configuration files under `/etc/ssh/sshd_config.d/`.

Root access is denied by default unless explicitly allowed by a configuration file.

Reply:
OK. I din't know that. I'm sorry.